src/main/java/ca/on/oicr/pineryreports/reports/impl/ProjectSequencingReport.java

```diff
@@ -0,0 +1,212 @@
+    List<SampleDto> samples = pinery.getSample().all();
+    allSamplesById = mapSamplesById(samples);
+    Map<Integer, UserDto> allUsersById = pinery.getUser().all().stream()
+        .collect(Collectors.toMap(UserDto::getId, Functions.identity()));
+    
+    List<ReportObject> rows = new ArrayList<>();
+    List<RunDto> allRuns = pinery.getSequencerRun().all();
+    for (RunDto run : allRuns) {
+      if ("Completed".equals(run.getState()) && run.getPositions() != null) {
+        for (RunDtoPosition pos : run.getPositions()) {
+          if (pos.getSamples() != null) {
+            for (RunDtoSample sam : pos.getSamples()) {
+              SampleDto dilution = allSamplesById.get(sam.getId());
+              if (project.equals(dilution.getProjectName())) {
+                UserDto poolCreator = pos.getPoolCreatedById() == null ? null : allUsersById.get(pos.getPoolCreatedById());
+                rows.add(new ReportObject(dilution, run, pos, poolCreator));
+              }
+            }
+          }
+        }
+      }
+    }
+    rows.sort(byPoolDate);
+    reportData = rows;
+  }
+  
+  /**
+   * Sort descending by pool date
+   */
+  private final Comparator<ReportObject> byPoolDate = (o1, o2) -> {
+    String o1Created = removeTime(o1.getLane().getPoolCreated());
+    String o2Created = removeTime(o2.getLane().getPoolCreated());
+    if (o1Created == null) {
+      if (o2Created != null) {
+        return -1;
+      }
+    } else if (o2Created == null) {
+      if (o1Created != null) {
+        return 1;
+      }
+    } else {
+      return o1Created.compareTo(o2Created) * -1;
+    }
+    return 0;
+  };
+
+  @Override
+  protected List<ColumnDefinition> getColumns() {
+    return COLUMNS;
+  }
+
+  @Override
+  protected int getRowCount() {
+    return reportData.size();
+  }
+
+  @Override
+  protected String[] getRow(int rowNum) {
+    ReportObject obj = reportData.get(rowNum);
+    String[] row = new String[COLUMNS.size()];
+    SampleDto stock = getParent(obj.getDilution(), "Stock", allSamplesById);
+    
+    String i1 = getUpstreamAttribute("Barcode", obj.getDilution(), allSamplesById);
+    String i2 = getUpstreamAttribute("Barcode Two", obj.getDilution(), allSamplesById);
+    String indices = i1 == null ? "" : i2 == null ? i1 : i1 + ", " + i2;
+    
+    int i = 0;
+    // Stock
+    row[i++] = stock.getName();
+    // Pool
+    row[i++] = obj.getLane().getPoolName();
+    // Pool Creator
+    row[i++] = getUserName(obj.getPoolCreator());
+    // Pool Created
+    row[i++] = obj.getLane().getPoolCreated() == null ? null : removeTime(obj.getLane().getPoolCreated());
+    // Dilutions
+    row[i++] = Integer.toString(obj.getLane().getSamples().size());
+    // Library
+    row[i++] = obj.getDilution().getName();
+    // Run
+    row[i++] = obj.getRun().getName();
+    // Lane
+    row[i++] = obj.getLane().getPosition().toString();
+    // Index
+    row[i++] = indices;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "i". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)